src/components/Collections/Collections.js

```diff
@@ -25,13 +43,43 @@ class Collections extends React.Component {
 
   render() {
     const { collections } = this.state;
+
     return (
       <>
-        {collections && collections.length === 0 && (
-          <div>This user does not have any collections.</div>
-        )}
+        <LoadingBar color="#6958f2" ref={this.loadingBar} />
```

Comment:
remind me about this sunday `<LoadingBar color="#6958f2" ref={this.loadingBar} />`